src/app/public/modules/datepicker/datepicker.component.ts

```diff
@@ -1,67 +1,83 @@
 import {
+  ChangeDetectionStrategy,
   Component,
   EventEmitter,
-  ViewChild
+  OnDestroy,
+  ViewChild,
+  ChangeDetectorRef
 } from '@angular/core';
 
-import { Subject } from 'rxjs/Subject';
-
-import { SkyDatepickerCalendarComponent } from './datepicker-calendar.component';
-
 import {
```

Comment:
Shouldn't our libraries be below `rxjs` in the import ordering.

Reply:
`@skyux` is on the same "level" as `rxjs`. I'm going with alphabetical order based on that.

Reply:
Ok. I have always went with Angular -> Third party libraries -> Blackbaud libraries -> internal things. But I also see your logic :) Not a huge deal.